### PR TITLE
fix(node): add @novu/nest deprecated warning and add novu/node self host info

### PR DESCRIPTION
**Pull request by jainpawan21**
### What changed? Why was the change needed?

1. Add `@novu/nest` deprecation warning
2. Add `@novu/node` self host config info 
### Screenshots

<details>
<summary><strong>Expand for optional sections</strong></summary>

### Special notes for your reviewer

</details>

### CODE REVIEW

**Review by SokratisVidros — APPROVED** [review]

File: packages/node/README.md [hunk 1]
```diff
@@ -62,9 +62,9 @@ yarn add @novu/node
 ```ts
 import { Novu } from '@novu/node';
 
-const novu = new Novu(process.env.NOVU_SECRET_KEY);
+const novu = new Novu('<NOVU_SECRET_KEY>');
```
Comment (rifont) [inline packages/node/README.md:65]: I think most folks will make use the previous of `process.env.NOVU_SECRET_KEY` pattern, this is the same pattern we follow elsewhere in our documentation. Was there a specific reason for this change?

**Review by rifont — APPROVED** [review]
🔥

File: packages/node/README.md (hunk 1 above)
Comment (jainpawan21) [reply packages/node/README.md:65]: Sorry I merge this PR before reading this comment. in this whole readme file, we were using `<NOVU_SECRET_KEY>`. So I replace process.env.NOVU_SECRET_KEY with <NOVU_SECRET_KEY>

Comment (rifont) [reply packages/node/README.md:65]: That makes sense for this PR, thanks for clarifying. We should align on a pattern everywhere in future docs updates. I prefer `new Novu(process.env.NOVU_SECRET_KEY);` as it's workable out of the box.

**Pull request merged**